packages/rancher-alerting-drivers/README.md

```diff
@@ -0,0 +1,10 @@
+# rancher-alerting-drivers
+
+To add a new sub chart:
+- develop the new sub chart as an independent chart under the `packages/` directory in this repo
+- add a new directory and dependency.yaml file under the `generated-changes/dependencies/` directory in the current directory
+- update the `charts/templates/values.yaml` file with the configuration for the new sub chart
+
+To develop this chart and sub charts, please follow the instructions for Rancher Chart and Package:
+- [Rancher Chart](https://github.com/rancher/charts/tree/master#rancher-chart-structure)
+- [Rancher Package](https://github.com/rancher/charts/tree/dev-v2.5-source/packages#packages)
```

Comment:
Let's use the term "Driver" instead of "sub chart" for consistent language

Reply:
fixed.